Core/src/main/java/com/plotsquared/core/command/Visit.java

```diff
@@ -214,61 +203,31 @@ private void visit(@NotNull final PlotPlayer player, @NotNull final PlotQuery qu
-                                // Try to parse a plot
-                                final Plot plot = MainUtil.getPlotFromString(player, finalArgs[0], true);
-                                if (plot == null) {
-                                    MainUtil.sendMessage(player, Captions.NOT_VALID_PLOT_ID);
-                                    return;
-                                }
-                                this.visit(player, PlotQuery.newQuery().withPlot(plot), null, confirm, whenDone, 1);
-                            }
+                            // player not found
+                            MainUtil.sendMessage(player, Captions.INVALID_PLAYER, finalArgs[0]);
                         } else {
                             this.visit(player, PlotQuery.newQuery().ownedBy(uuid).whereBasePlot(), null, confirm, whenDone, finalPage);
                         }
                     });
                 } else {
-                    if (finalPage == Integer.MIN_VALUE && MathMan.isInteger(finalArgs[0])) {
-                        // The argument was a number, so we assume it's the page number
-                        int parsedPage;
-                        try {
-                            parsedPage = Integer.parseInt(finalArgs[0]);
-                            this.visit(player, PlotQuery.newQuery().ownedBy(player).whereBasePlot(), null, confirm,
-                                whenDone, parsedPage);
-                        } catch (final Throwable throwable) {
-                            MainUtil.sendMessage(player, Captions.NOT_A_NUMBER, finalArgs[0]);
-                        }
-                    } else {
-                        // Try to parse a plot
-                        final Plot plot = MainUtil.getPlotFromString(player, finalArgs[0], true);
-                        if (plot != null) {
-                            this.visit(player, PlotQuery.newQuery().withPlot(plot), null, confirm, whenDone, 1);
-                        }
+                    // Try to parse a plot
+                    final Plot plot = MainUtil.getPlotFromString(player, finalArgs[0], true);
+                    if (plot != null) {
+                        this.visit(player, PlotQuery.newQuery().withPlot(plot), null, confirm, whenDone, 1);
                     }
                 }
                 break;
             case 0:
-                // /p v
-                this.visit(player, PlotQuery.newQuery().ownedBy(player), null, confirm, whenDone);
-                break;
+                // /p v is invalid
+                Captions.COMMAND_SYNTAX.send(player, getUsage());
+                return CompletableFuture.completedFuture(false);
             default:
         }
 
         return CompletableFuture.completedFuture(true);
     }
 
-    public Collection<Command> tab(PlotPlayer player, String[] args, boolean space) {
+    @Override public Collection<Command> tab(PlotPlayer player, String[] args, boolean space) {
```

Comment:
in `case 0` it still completes numbers, this should only be done for `/plot home`. `case 1` is also wrong as it assumes a number for the first argument, which is only the case for home, so the entire tab completion here is wonky